From c205f25826f2d0a4ad6dc8cbc7564c5cb7d6003b Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 6 Jan 2016 12:53:23 +0000
Subject: [PATCH 0711/1076] MANTIS 39707 Mantis 39707: For Gatwick allow D1
 mode in ESTOP situation - extend ConveyorDev.idl subdevice types list by
 BELT. Thus it would be possible to tell BELTs are down.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@942 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
index 07ace0a6..f059ca5d 100644
--- a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
@@ -39,7 +39,7 @@
 **/
 module conveyordev
 {
-   const string interface_version = "1.6";
+   const string interface_version = "1.7";
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
   RC indicationListener::indication(in any ind);
@@ -75,7 +75,8 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
      RFID_READER,         // get data from rfid-tags on baggage
      RFID_WRITER,         // set data to rfid-tags on baggage
      IMAGE_SCANNER,       // get image of baggage
-     SAFETY_DOOR          // prevent unauthorized access
+     SAFETY_DOOR,         // prevent unauthorized access
+     BELT                 // conveyor belt, motors, and gears
    };
 
    struct moduleCharacteristic
-- 
2.41.0.windows.1

